Source/WebCore/animation/TimelineRange.cpp

```diff
@@ -43,11 +48,125 @@ bool SingleTimelineRange::isDefault(const CSSPrimitiveValue& value, Type type)
     return value.valueID() == CSSValueNormal || (value.isPercentage() && !value.isCalculated() && percentIsDefault(value.resolveAsPercentageNoConversionDataRequired(), type));
 }
 
+Length SingleTimelineRange::defaultValue(Type type)
+{
+    return type == Type::Start ? Length(0, LengthType::Percent) : Length(100, LengthType::Percent);
```

Comment:
The duplication between this and `percentIsDefault` would be nice to avoid.

Reply:
Fixed.